### ISSUE TITLE
Ints are being turned into Uids and vice versa when both are present in a plist.

### ISSUE THREAD

**Issue by Redoubts** [wooster/biplist#9]
I'm noticing confusion in how Uids are being handled, when a plist is going to contain both UIds and Ints. Example:

```
# int to Uid
a = {'$archiver': 'NSKeyedArchiver',
 '$objects': ['$null',
  {'$class': Uid(3), 'somekey': Uid(2), 'someOtherKey': 3},
  'object value as string',
  {'$classes': ['Archived', 'NSObject'], '$classname': 'Archived'}],
 '$top': {'root': Uid(1)},
 '$version': 100000}

biplist.readPlistFromString(biplist.writePlistToString(a))

{'$archiver': 'NSKeyedArchiver',
 '$objects': ['$null',
  {'$class': Uid(3), 

'someOtherKey': Uid(3), 

'somekey': Uid(2)},
  'object value as string',
  {'$classes': ['Archived', 'NSObject'], '$classname': 'Archived'}],
 '$top': {'root': Uid(1)},
 '$version': 100000}

# Uid to int

a = {'$archiver': 'NSKeyedArchiver',
 '$objects': ['$null',
  {'$class': 1, 'somekey': 2},
  'object value as string',
  {'$classes': ['Archived', 'NSObject'], '$classname': 'Archived'}],
 '$top': {'root': Uid(1)},
 '$version': 100000}

biplist.readPlistFromString(biplist.writePlistToString(a))

{'$archiver': 'NSKeyedArchiver',
 '$objects': ['$null',
  {'$class': 1, 'somekey': 2},
  'object value as string',
  {'$classes': ['Archived', 'NSObject'], '$classname': 'Archived'}],

 '$top': {'root': 1},

 '$version': 100000}
```

```
OS X 10.11
Python 2.7.10 (default, Oct 23 2015, 18:05:06) 
IPython 4.0.0 -- An enhanced Interactive Python.
```

**Comment by Redoubts** [comment]
Even more simply:

```
In [65]: biplist.readPlistFromString(biplist.writePlistToString(
    [1, biplist.Uid(1)]
))
Out[65]: [1, 1]

In [66]: biplist.readPlistFromString(biplist.writePlistToString(
    [biplist.Uid(1), 1]
))
Out[66]: [Uid(1), Uid(1)]

```

**Comment by wooster** [comment]
This should be fixed in the 1.0.0 release, out today. Thanks!
